test/.eslintrc

```diff
@@ -0,0 +1,5 @@
+{
+  "env": {
+    "mocha": true
+  }
+}
```

Comment:
To compare [ESLint repo](https://github.com/eslint/eslint/tree/master/tests/lib) use a `.eslintrc` in their test folder.

Env mocha will adds all of the Mocha testing global variables. https://github.com/sindresorhus/globals/blob/master/globals.json#L683